minet/buzzsumo/types.py

```diff
@@ -0,0 +1,144 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+from casanova import TabularRecord
+
+
+@dataclass
+class BuzzsumoArticle(TabularRecord):
+    buzzsumo_id: Optional[str]
+    twitter_shares: Optional[int]
+    facebook_shares: Optional[int]
+    url: Optional[str]
+    updated_at: Optional[str]
+    facebook_likes: Optional[int]
+    total_facebook_shares: Optional[int]
+    title: Optional[str]
+    published_date: Optional[str]
+    pinterest_shares: Optional[int]
+    og_url: Optional[str]
+    article_amplifiers: Optional[str]
+    domain_name: Optional[str]
+    author_name: Optional[str]
+    num_words: Optional[int]
+    subdomain: Optional[str]
+    twitter_user_id: Optional[str]
+    video_length: Optional[str]
+    language: Optional[str]
+    alexa_rank: Optional[str]
```

Comment:
Is this really a str?

Reply:
Same for `youtube_trending_score`, `evergreen_score` etc.

Reply:
Changed : 
`evergreen_score` (str -> float), `evergreen_score2` (str -> float), `alexa_rank` (str -> int), `video_length` (str -> int), `twitter_user_id` (str -> int), `published_date` (str -> int), `updated_at` (str -> int)